system/Filters/Filters.php

```diff
@@ -329,42 +315,50 @@ public function addFilter(string $class, string $alias = null, string $when = 'b
 	 * after the filter name, followed by a comma-separated list of arguments that
 	 * are passed to the filter when executed.
 	 *
-	 * @param string $name
+	 * @param string|array $filter
 	 * @param string $when
 	 *
 	 * @return Filters
 	 */
-	public function enableFilter(string $name, string $when = 'before')
+	public function enableFilter($filter, string $when = 'before')
```

Comment:
this is BC break ref https://3v4l.org/7b0GA

Reply:
> besides to revert cosmetic changes, please provide unit test

I don't know how to make a unit test for this change